src/conmon.c

```diff
@@ -534,10 +534,14 @@ static int write_oom_files()
 	return oom_fd >= 0 ? 0 : -1;
 }
 
-static gboolean oom_cb_cgroup_v1(int fd, GIOCondition condition, G_GNUC_UNUSED gpointer user_data)
+// user_data is expected to be the container's pid, used to verify the cgroup hasn't been cleaned up
+static gboolean oom_cb_cgroup_v1(int fd, GIOCondition condition, gpointer user_data)
 {
+	int *pid = (int *)user_data;
```

Comment:
should there be a null check here? I considered it but wasn't sure what to do with a null value...